ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/common/CoreEnvironmentBean.java

```diff
@@ -48,6 +48,8 @@ public class CoreEnvironmentBean implements CoreEnvironmentBeanMBean
     @FullPropertyName(name = "timeout.factor")
     private volatile int timeoutFactor = 1;
 
+    private boolean logAndRethrow = true;
```

Comment:
@mmusgrov  Could we have configurable property for the logAndRethrow too?

Reply:
We use beans now so, for example, setting it via the properties file is:

```
<entry key="CoreEnvironmentBean.logAndRethrow">true</entry>
```

or via a system property `-DCoreEnvironmentBean.logAndRethrow=false`

The method you cite is the old one and we don't want to encourage it.

Reply:
Ok, thanks @mmusgrov for clarifying it.